README.md

```diff
@@ -93,92 +93,170 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.90 - March 2025 Update
+### 0.91 - May 2025 Update
 
 In this release, we focused on new features, stability, and automation.
 
 **✨Highlights**
 
-![Gif for Command Palette](doc/images/overview/CmdPal_Hero.gif)
+ - Added open URL fallback command for the WebSearch extension, enabling users to directly open URLs in the browser from Command Palette.
```

Comment:
Highlights need to be upleveled here.  this is too deep

Reply:
I just did the top 4 items here, feel free to take a look